### PR TITLE
Empty state not queryable layers

### PR DESCRIPTION
**Pull request by kepatinin**
- Added empty state when layer does not have queryable sublayers
- Infobox results are buttons only if they have results
- Make style more similar for polygon results and layers results

### CODE REVIEW

**Comment by artsdatabank** [comment]
Test den: https://okologiskegrunnkart.test.artsdatabanken.no/pr/413

File: src/Okologiskegrunnkart/FeatureInfo/PolygonElement.js [hunk 1]
```diff
@@ -44,7 +44,7 @@ const PolygonElement = ({
     <div className="generic_element">
       <ListItem
         id="polygon-element-list"
-        button={numberResults > 0 ? true : false}
```
Comment (bjornreppen) [inline src/Okologiskegrunnkart/FeatureInfo/PolygonElement.js:47]: A bit of redundancy here mapping true to true and false to false

**Review by bjornreppen — APPROVED** [review]
Good job

File: src/Okologiskegrunnkart/FeatureInfo/PolygonElement.js (hunk 1 above)
Comment (kepatinin) [reply src/Okologiskegrunnkart/FeatureInfo/PolygonElement.js:47]: Totally agree, that's why I removed it :D replaced it with `button={numberResults}`

**Pull request merged**